hack/extract-test-inputs

```diff
@@ -12,6 +12,7 @@ declare -ra ACCESSIONS=(
   ENCBS873XSG
   ENCBS051RNA
   ENCBS393JAH
+  ENCBS630AAA
```

Comment:
Adding an accession to the test inputs here so that we can include examples of antibodies that don't have targets.